Makefile

```diff
@@ -102,7 +102,15 @@ helm-package:
 	helm package config/ks-core --app-version=v3.1.0 --version=0.1.0 -d ./bin
 
 helm-deploy:
+	ls config/crds/ | grep -v types.kubefed.io | xargs -i cp -r config/crds/{} config/ks-core/crds/
```

Comment:
`grep -v types.kubefed.io` can be removed now, since we have deleted all the CRDs from config/crds/ folder.

Reply:
Ok. I will also remove it from `make package`.